gui/default/assets/lang/lang-en.json

```diff
@@ -335,10 +345,13 @@
    "Unavailable": "Unavailable",
    "Unavailable/Disabled by administrator or maintainer": "Unavailable/Disabled by administrator or maintainer",
    "Undecided (will prompt)": "Undecided (will prompt)",
+   "Unencrypted Items": "Unencrypted Items",
```

Comment:
Again, we don't know if they are encrypted or not, might just be the wrong password.